neurom/fst/tests/test_neuritefunc.py

```diff
@@ -213,6 +213,12 @@ def test_section_branch_orders():
                     (0, 1, 1,  # type 3, basal dendrite
                      0, 1, 1)) # type 2, axon
 
+def test_section_strahler_orders():
+    strahler_orders = list(_nf.section_strahler_orders(SIMPLE))
+    assert_allclose(strahler_orders,
+                    (2, 1, 1,  # type 3, basal dendrite
```

Comment:
The definition of this ordering is not consistent (see below). I will try to find the original paper for a proper reference.

Neurolucida definition:
```
    Order assignment progresses towards the root segment.
    At each node, if **all daughter** segments are assigned the same order, the parent segment is assigned the daughter segments order "+1."
    If the daughter segments are assigned different orders, the largest order is assigned to the parent segment.
```
Wiki definition:
```
    If the node is a leaf (has no children), its Strahler number is one.
    If the node has one child with Strahler number i, and all other children have Strahler numbers less than i, then the Strahler number of the node is i again.
    If the node has **two or more children** with Strahler number i, and no children with greater number, then the Strahler number of the node is i + 1.
```